components/ActivitiesList/ActivitiesList.js

```diff
@@ -59,8 +59,12 @@ export default function ActivitiesList({ handleShowForm }) {
           <Styled.Card
             onClick={handleShowForm}
```

Comment:
if the handleShowForm function ist declared in another component, you should rename it to onShowForm when passing it down to adhere to our naming convention